doc/cli.md

```diff
@@ -0,0 +1,230 @@
+
+The `validate` subcommand validates a single Statement against Statement Templates in one or more Profiles. The following table shows its arguments:
+
+| Subcommand&nbsp;Argument | Description
+| :--                          | :--
+| `-p, --profile URI`          | Profile URI filepath/location; must specify one or more.
+| `-i, --template-id IRI`      | IDs of Statement Templates to validate against; can specify zero or more. Filters out all Templates that are not included.
+| `-s, --statement URI`        | Statement filepath/location; must specify one.
+| `-e, --extra-statements URI` | Extra Statement batch filepath/location; can specify zero or more. If specified, activates [Statement Ref property](library.md#statement-ref-templates) validation, where the referred object/context Statement exists in this batch and its Template exists in a provided Profile.
+| `-a, --all-valid`            | If set, the Statement is not considered valid unless it is valid against ALL Templates. Otherwise, it only needs to be valid against at least one Template.
+| `-c, --short-circuit`        | If set, then print on only the first Template the Statement fails validation against.Otherwise, print for all Templates the Statement fails against.
+| `-h, --help`                 | Display the help menu.
+
+The `match` subcommand matches a Statement batch against Patterns in one or more Profiles. The following table shows its arguments:
+
+| Subcommand&nbsp;Argument | Description
+| :--                    | :--
+| `-p, --profile URI`    | Profile filepath/location; must specify one or more.
+| `-i, --pattern-id IRI` | IDs of primary Patterns to match against; can specify zero or more. Filters out all Patterns that are not included.
+| `-s, --statement URI`  | Statement filepath/location; must specify one or more.
+| `-n, --compile-nfa`    | If set, compiles the Patterns into a non-deterministic finite automaton (NFA) instead of a deterministic one, allowing for more detailed error traces at the cost of decreased performance.
+| `-h, --help`           | Display the help menu.
+
+## Examples for `persephone validate`
+
+In the examples in this and the next section, assume that the `test-resources/sample_profiles` and `test-resources/sample_statements` directories were copied into `target/bundle`, i.e. the location where we run `./bin/persephone`.
+
+For our first example, we can validate the `calibration_1.json` Statement against the Statement Templates in the `calibration.jsonld` Profile:
+```
+% ./bin/persephone.sh validate \
+  --profile sample_profiles/calibration.jsonld \
+  --statement sample_statements/calibration_1.json
+```
+This will run, print nothing, and exit with code 0, indicating successful validation.
+
+We can also add additional Profiles as so:
+```
+% ./bin/persephone.sh validate \
+  --profile sample_profiles/calibration.jsonld \
+  --profile test-resources/sample_profiles/catch.json \
```

Comment:
Path inconsistency?

Reply:
OOPS

Reply:
That's what you (I?) get for blind copy-pasting

Reply:
A small price to pay for nice, detailed docs like this!

Reply:
d30cb2aaf064edcfcde206c9fd2aa262792acbde